tests/Browser/OrdersTest.php

```diff
@@ -25,4 +25,21 @@ public function test_user_can_see_orders()
                 ->waitForText(text: 'Order Type -> Name', seconds: 30);
         });
     }
+
+    // test for deleting order
+    public function test_user_can_delete_open_order()
```

Comment:
I think this is not finished?

Reply:
this doesent belong to this branch - how it ended up here - maybe i merged test - browser branch